### PR TITLE
Doro's ores

### PR DESCRIPTION
**Pull request by KingBalDoro**

### CODE REVIEW

File: src/main/java/com/mraof/minestuck/world/lands/terrain/FungiLandType.java [hunk 1]
```diff
@@ -103,12 +103,13 @@ public void addBiomeGeneration(LandBiomeGenBuilder builder, StructureBlockRegist
...
-						CountPlacement.of(48), InSquarePlacement.spread(), HeightRangePlacement.uniform(VerticalAnchor.bottom(), VerticalAnchor.absolute(64)), BiomeFilter.biome()),
+						new OreConfiguration(blocks.getGroundType(), MSBlocks.MYCELIUM_STONE_CRUXITE_ORE.get().defaultBlockState(), 4),
+						CountPlacement.of(24), InSquarePlacement.spread(), HeightRangePlacement.uniform(VerticalAnchor.bottom(), VerticalAnchor.absolute(64)), BiomeFilter.biome()),
 				LandBiomeType.any());
+		
 		builder.addFeature(GenerationStep.Decoration.UNDERGROUND_ORES, PlacementUtils.inlinePlaced(Feature.ORE,
-						new OreConfiguration(blocks.getGroundType(), Blocks.REDSTONE_ORE.defaultBlockState(), 8),
-						CountPlacement.of(36), InSquarePlacement.spread(), HeightRangePlacement.uniform(VerticalAnchor.bottom(), VerticalAnchor.absolute(32)), BiomeFilter.biome()),
+						new OreConfiguration(blocks.getGroundType(), MSBlocks.MYCELIUM_STONE_URANIUM_ORE.get().defaultBlockState(), 2),
+						CountPlacement.of(12), InSquarePlacement.spread(), HeightRangePlacement.uniform(VerticalAnchor.bottom(), VerticalAnchor.absolute(32)), BiomeFilter.biome()),
```
Comment (Dweblenod) [inline src/main/java/com/mraof/minestuck/world/lands/terrain/FungiLandType.java:112]: I gave bad instructions. Delete the two Ore Features then add the two following lines of code around line 71 registry.setBlock("cruxite_ore", MSBlocks.MYCELIUM_STONE_CRUXITE_ORE);
registry.setBlock("uranium_ore", MSBlocks.MYCELIUM_STONE_URANIUM_ORE);

File: src/main/java/com/mraof/minestuck/item/MSItems.java [hunk 2]
```diff
@@ -668,6 +668,8 @@
 	public static final RegistryObject<BlockItem> STEEL_BEAM = registerBlockItem(MSBlocks.STEEL_BEAM, MSItemGroup.LANDS);
 	public static final RegistryObject<BlockItem> MYCELIUM_COBBLESTONE = registerBlockItem(MSBlocks.MYCELIUM_COBBLESTONE, MSItemGroup.LANDS);
 	public static final RegistryObject<BlockItem> MYCELIUM_STONE = registerBlockItem(MSBlocks.MYCELIUM_STONE, MSItemGroup.LANDS);
+	public static final RegistryObject<BlockItem> MYCELIUM_STONE_CRUXITE_ORE = registerBlockItem(MSBlocks.MYCELIUM_STONE_CRUXITE_ORE, MSItemGroup.LANDS);
+	public static final RegistryObject<BlockItem> MYCELIUM_STONE_URANIUM_ORE = registerBlockItem(MSBlocks.MYCELIUM_STONE_URANIUM_ORE, MSItemGroup.LANDS);
```
Comment (Dweblenod) [inline src/main/java/com/mraof/minestuck/item/MSItems.java:672]: Move this code to a line next to the other uranium and cruxite ores(respectively)

**Comment by Dweblenod** [comment]
Only the pink stone coal ore needed retexturing, there are minute changes to the other pink stone ores that have been reverted in this Pull Request

**Comment by KingBalDoro** [comment]
check this

**Comment by KingBalDoro** [comment]
do not check this

**Comment by KingBalDoro** [comment]
check this

**Review by Dweblenod — COMMENTED** [review]
Looks good in game and in code except for the little mixup in MSBlocks and the pink stone ore textures! The pink stone coal ore obviously is a valid change, but the changes to the other pink stone ores have caused them to revert to a previous state, the pink stone uranium ore especially as it now uses the old uranium template

**Comment by KingBalDoro** [comment]
i'm not sure how to revert the changes to the rest of the coal so i'll just delete all the other ones we don't need

**Comment by Dweblenod** [comment]
> i'm not sure how to revert the changes to the rest of the coal so i'll just delete all the other ones we don't need

If they are deleted, then there will no longer be textures for those blocks. Sending options for reversing the changes over discord

**Comment by KingBalDoro** [comment]
check this

**Comment by Dweblenod** [comment]
Looks good, nice work!

**Pull request merged**